nowcasting_dataset/dataset/datasets.py

```diff
@@ -429,12 +408,8 @@ def subselect_data(
     )
 
     # t0_dt or if not available use a different datetime index
-    if T0_DT in batch.keys():
-        current_time_of_first_batch = batch[T0_DT][0]
-    else:
-        current_time_of_first_batch = batch[date_time_index_to_use].isel(
-            time=current_timestep_index
-        )[0]
+    batch.general.to_numpy()
```

Comment:
Is there a way we could keep it as xarrays? changing it back and forth from NumPy adds some overhead to it.

Reply:
Ive changed it so it just takes .value to t0_dt